packages/e2e-tests/specs/widgets/adding-widgets.test.js

```diff
@@ -85,18 +116,18 @@ describe( 'Widgets screen', () => {
 	*/
 
 	async function getInlineInserterButton() {
-		return await page.waitForSelector(
-			'button[aria-label="Add block"][aria-haspopup="true"]',
-			{
-				visible: true,
-			}
-		);
+		return await find( {
+			role: 'combobox',
+			name: 'Add block',
+			haspopup: 'menu',
```

Comment:
Is it fine to omit `haspopup`? In my mind it doesn't make much sense to make the test rely on that being there.

Reply:
Oh yeah probably! It makes sense to omit this.